lib/xmlenc.js

```diff
@@ -182,15 +182,10 @@ function decryptKeyInfo(doc, options) {
 }
 
 function decryptKeyInfoWithScheme(encryptedKey, options, scheme) {
```

Comment:
This is the function I was concerned could be problematic in Node 6. This change is not strictly necessary, I just happened to notice it whilst reviewing it.